pkgs/development/python-modules/django-jsonfield/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, lib, buildPythonPackage, isPyPy, fetchPypi
+, django, sqlite }:
+
+buildPythonPackage rec {
+  pname = "django-jsonfield";
+  version = "1.0.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0gna25jizsk46d367ygzh9ba9hvcjv7qbql6bnsnb4vkaiazs2kc";
+  };
+
+  preCheck = ''
+    export DB_ENGINE=sqlite3
+    export DB_NAME=tests
+  '';
+
+  checkInputs = [ django sqlite ];
+
+  # one test is failing :(
```

Comment:
If you can maybe find out the reason as to why it fails and also report it upstream,
it would be acceptable to just patch the test out or remove it.